addons/l10n_ro_efactura/models/ciusro_document.py

```diff
@@ -164,19 +164,26 @@ def _request_ciusro_download_answer(self, company, key_download, session):
 
         # E-Factura gives download response in ZIP format
         zip_ref = zipfile.ZipFile(io.BytesIO(result['content']))
-        signature_file = next(file for file in zip_ref.namelist() if 'semnatura' in file)
-        xml_bytes = zip_ref.open(signature_file)
+        # The ZIP will contain two files, one with the original invoice or with the identified errors (as the case may be)
+        # and the other with the electronic signature (containing 'semnatura').
+        # If there is an error (status == 'nok') we want to provide the file with the errors.
+        if status == 'nok':
+            provided_file = next(file for file in zip_ref.namelist() if 'semnatura' not in file)
+        else:
+            provided_file = next(file for file in zip_ref.namelist() if 'semnatura' in file)
+        xml_bytes = zip_ref.open(provided_file)
```

Comment:
could you try something quickly ? Rename provided_file to signature_file like it was before, so that the xml_bytes line is untouched ? It should make the security check disappeaaaaar

Reply:
All good now, how should we proceed? I'm guessing we let the security people known and change back to provided_file after they give their okay?

Reply:
For me we can let signature_file so that we don't have to ping the security team and wait for qdp for the final review :smile:

Reply:
But when there's an error we choose the error file and not the signature file 🙈

Reply:
It's just a naming of variable, i'm fine with both, either rename and ping the security team or let it like this

Reply:
I'll go ahead and rename it to avoid having an inaccurate variable name.